switchtool/switch/switch.py

```diff
@@ -155,7 +158,12 @@ def power(self):
         return self._power
 
     def get_enablepw(self):
-        pass
+        """
+        Placeholder for child class
+
+        This should request enablepw from user and set it as self._enablepw
+        """
+        ...
```

Comment:
If this is meant to be overloaded, we could raise `NotImplementedError` here to be extra clear